one_fm/operations/doctype/process_task/process_task.py

```diff
@@ -275,7 +275,7 @@ def create_tasks_for(tasks_list):
 def run_daily_process_task():
 	try:
 		"""Trigger all the Task creating process tasks for the day for all non Cron processes"""
-		today = 	datetime.today()
+		today = 	frappe.utils.get_datetime()
```

Comment:
We imported the now_datetime, Use the same here
https://github.com/ONE-F-M/one_fm/blob/cc40697e0a7e2a5980929f96b13172d3d78ac186/one_fm/operations/doctype/process_task/process_task.py#L17

Reply:
updated @pjamsheer